### PR TITLE
gcloud-sql-backups-describe: add page

### PR DESCRIPTION
**Pull request by giladAlboher**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/gcloud-sql-backups-describe.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# gcloud sql backups describe
+
+> Retrieves information about a backup.
+> More information: <https://cloud.google.com/sdk/gcloud/reference/sql/backups/describe>.
+
+- Code description:
+
+`gcloud sql backups describe ID --instance=INSTANCE, -i INSTANCE [GCLOUD_WIDE_FLAG …]`
```
[the remark this answers is not in the document]
Comment (giladAlboher) [reply pages/common/gcloud-sql-backups-describe.md]: Ok, let me know if its good

File: pages/common/gcloud-sql-backups-describe.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# gcloud sql backups describe
+
+> Retrieves information about a backup.
+> More information: <https://cloud.google.com/sdk/gcloud/reference/sql/backups/describe>.
+
+- Code description:
```
Comment (kbdharun) [inline pages/common/gcloud-sql-backups-describe.md:6]:
```suggestion
> Retrieve information about a backup.
> See also: `gcloud`.
> More information: <https://cloud.google.com/sdk/gcloud/reference/sql/backups/describe>.

- Retrieve information about a backup:
```

We can update the description to match the documentation and reference the main page.

File: pages/common/gcloud-sql-backups-describe.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+`gcloud sql backups describe {{backup-id}} --instance={{instance-id}}{{GCLOUD_WIDE_FLAG …}}`
```
Comment (kbdharun) [inline pages/common/gcloud-sql-backups-describe.md:8]:
```suggestion
`gcloud sql backups describe {{backup_id}} --instance={{instance_id}}`
```
We can use the snake case here and I don't think we need to specify the Gcloud-wide flag.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**